docs/source/workflows/observe/observe-workflow-with-dynatrace.md

```diff
@@ -0,0 +1,122 @@
+# Observing a Workflow with Dynatrace
```

Comment:
This needs the full Apache copyright header. 
Running `python ci/scripts/copyright.py --fix-all` should do this for you.

Reply:
thank you.  I ran that and it added the header.

```
INFO:File: docs/source/workflows/observe/observe-workflow-with-dynatrace.md. Changing line(s) 1
INFO:Copyright check passed
```